pkg/importer/openapi.go

```diff
@@ -349,3 +353,32 @@ func (l *loader) buildParam(p *openapi3.Parameter) Param {
 		In: p.In,
 	}
 }
+
+func hasToBeGoSafe(in string) bool {
+	return strings.ToLower(in) == "query"
+}
+
+func convertToGoSafe(name string) string {
```

Comment:
this func is a bit gross to be honest. You could probably make it much simpler by doing `strings.Split(name, "-")` and then rejoining with some smarts.

Reply:
i thought of that, but i think with the current function we can support more unsafe characters later if needed, what do you think?

Reply:
its fine